doc-content/enterprise-only/optimizer/spring-boot-ref-download-proc.adoc

```diff
@@ -0,0 +1,31 @@
+[id='spring-boot-ref-download-proc_{context}']
+
+= Downloading and building the school timetable reference implementation
+
+If you want to see a completed example of the school timetable project for {PLANNER} with Spring Boot product, download the starter application from the Red Hat Customer Portal.
+
+.Procedure
+. Navigate to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required), and select the product and version from the drop-down options:
+
+* Product: {PRODUCT}
+* Version: {PRODUCT_VERSION}
+. Download *{PRODUCT} {PRODUCT_VERSION} Reference Implementations* (`{PRODUCT_FILE}-reference-implementation.zip`).
+. Download *{PRODUCT} {PRODUCT_VERSION} Maven Repository* (`{PRODUCT_FILE}-maven-repository.zip`).
+. Extract the `{PRODUCT_FILE}-maven-repository.zip` file.
+. Copy the contents of the `{PRODUCT_FILE}-maven-repository/maven-repository` subdirectory into the `~/.m2/repository` directory.
+. Extract the `{PRODUCT_FILE}-reference-implementation.zip` file. This archive contains reference implementation ZIP files.
+. Extract the `{PRODUCT_FILE}-reference-implementation/{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-optaplanner-spring-boot-school-timetabling.zip` file.
+. Navigate to the `optaplanner-spring-boot-school-{MAVEN_ARTIFACT_VERSION}` directory.
+. Enter the following command to build the Spring Boot school timetabling project:
++
+[source]
+----
+mvn clean install -DskipTests
+----
+
+. Enter the following command to run the Spring Boot school timetabling project
```

Comment:
If the project uses a web UI, you also need to point a browser somewhere to access it? Alternatively if there is no UI and one needs to use curl as descreibed in chapter 27, perhaps you need to point the user to chapter 27?

Reply:
@adupliak can you clarify this step? Do we need to give the user a URL or point to the instructions to use curl?

Reply:
That is true. The UI is available by http://localhost:8080/ Could you please add this?